.github/workflows/my-ci-pipeline.yml

```diff
@@ -0,0 +1,15 @@
+name: Continuous Integration
+on: 
+  push:
+  pull_request:
+    paths:
+      - 'README.md'
```

Comment:
I think this isn't quite doing what you want! You're using `paths` rather than `paths-ignore` so currently PRs only trigger your tests if they involve changes to the Readme. That's not super important here because each push is triggering a run anyway, but worth noting

Potentially also worth noting that I _think_ (though not certain, so potentially worth testing!) that these statements don't "fall through", so on `push` here it won't pay attention to the README.md path at all here; if we want the same behaviour as on PR update then we need to repeat that block, e.g. like the below (except we probably want to change paths as per the above!)
```yml
  push:
    paths:
      - 'README.md'
  pull_request:
    paths:
      - 'README.md'
```